### PR TITLE
smoke-test centroids + graph overlay layer

### PR DESCRIPTION
**Pull request by seve**
This PR adds two tests to verify that centroid labels are generated/displayed and that the graph overlay layer correctly transforms the labels

### CODE REVIEW

**Comment by mweiden** [comment]
@seve you may need to rebase your branch against master and resolve merge conflicts ⬇️

File: client/__tests__/e2e/e2e.test.js [hunk 1]
```diff
@@ -309,6 +309,58 @@ describe("ui elements don't error", () => {
...
+  });
+});
+
+describe("centroid labels", () => {
+  test("labels are created", async () => {
+    await utils.clickOn("centroid-label-toggle");
+    const labels = Object.keys(data.categorical);
+    /* eslint-disable no-await-in-loop */
+    // Toggle colorby for each category and check to see if labels are generated
+    for (let i = 0, { length } = labels; i < length; i += 1) {
```
Comment (mweiden) [inline client/__tests__/e2e/e2e.test.js]: nit: I think here you can use a [`for-of`](https://stackoverflow.com/questions/3010840/loop-through-an-array-in-javascript) statement and make things a little cleaner

File: client/__tests__/e2e/puppeteerUtils.js [hunk 2]
```diff
@@ -231,53 +256,82 @@ export const cellxgeneActions = puppeteerPage => ({
...
+    await puppeteerUtils(puppeteerPage).clickOn(
+      `${categoryName}:delete-category`
+    );
   },
 
   async createLabel(categoryName, labelName) {
     await puppeteerUtils(puppeteerPage).clickOn(`${categoryName}:see-actions`);
-    await puppeteerUtils(puppeteerPage).clickOn(`${categoryName}:add-new-label-to-category`);
-    await puppeteerUtils(puppeteerPage).typeInto(`${categoryName}:new-label-name`, labelName);
+    await puppeteerUtils(puppeteerPage).clickOn(
```
Comment (mweiden) [inline client/__tests__/e2e/puppeteerUtils.js]: What line width are you using in Prettier? In the python code, we've been using 120 characters.

Comment (seve) [reply client/__tests__/e2e/puppeteerUtils.js]: AFAIK we use the default prettier settings which I see as 80.  Maybe in the future we can add a prettier config file

File: client/__tests__/e2e/e2e.test.js (hunk 1 above)
Comment (seve) [reply client/__tests__/e2e/e2e.test.js]: IIRC: `for of`'s don't work nicely with async/await

Comment (bkmartinjr) [reply client/__tests__/e2e/e2e.test.js]: they are also heinously slow.   We strongly dislike for..or in any situation where performance matters even a bit.

This code matches our current conventions.

File: client/__tests__/e2e/puppeteerUtils.js (hunk 2 above)
Comment (bkmartinjr) [reply client/__tests__/e2e/puppeteerUtils.js]: AFAIK, we use the default prettier config (i,e, we implicitly have a config - the default).   I believe 80 is the default line length.

File: client/__tests__/e2e/e2e.test.js (hunk 1 above)
Comment (mweiden) [reply client/__tests__/e2e/e2e.test.js]: Ah! Ok, good to know.

**Pull request merged**